.cache/v/cache/lastfailed

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
nothing in this folder should be in your repository.  You should ensure that this file is removed from your repository and then update .gitignore to ignore it when it is recreated